### PR TITLE
Mateo

### PR DESCRIPTION
**Pull request by Mateo-Wallace**
Added html file

### CODE REVIEW

**Review by SteveMustanski — CHANGES_REQUESTED** [review]
Hey, did you forget to but the h1 like you said you needed to do?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+
```
Comment (SteveMustanski) [inline index.html]: Did you mean to add the h1?

Comment (Mateo-Wallace) [reply index.html]: I fixed it nerd

**Review by SteveMustanski — APPROVED** [review]
LGTM

**Pull request merged**